src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -12,10 +15,128 @@
 from proto.ssl_gc_geometry_pb2 import Vector2
 
 # TODO (#2599): Remove Duration parameter from test
+
+
 @pytest.mark.parametrize(
-    "run_enemy_ai,test_duration", [(False, 20)]
+    "run_enemy_ai,test_duration", [(False, 20), (False, 20)]
```

Comment:
Little bit of technical debt that would be nice to solve here: Maybe we can set a fixed timeout, and remove it from the parameter list?

Reply:
resolved. Since a new referee command will be sent, I instead have a new method right below it. Would you recommend adding another parameter next_referee_command so it's possible to parametrize it?